src/couch/src/couch_server.erl

```diff
@@ -335,7 +335,10 @@ terminate(Reason, Srv) ->
     ok.
 
 handle_config_change("couchdb", "database_dir", _, _, N) ->
-    exit(couch_server(N), config_change),
+    case whereis(couch_server(N)) of
+        Pid when is_pid(Pid) -> exit(Pid, {shutdown, config_change});
+        undefined -> ok
+    end,
```

Comment:
When I ran `make eunit apps=couch` locally, I noticed that these lines don't seem to be covered. Do you think it's worthwhile to add some?

Reply:
The Pid one is covered in the chttpd test: `make eunit apps=chttpd suites=chttpd_dbs_info_test`
The undefined one might be tricky to cover as we'd have to kill the process then call whereis/1 on it.

Reply:
Hmm, I guess it doesn't show up in `src/couch/.eunit/couch_server.COVER.html` because it's a different OTP application? In any case, I believe you that it cleans up test log files, and am happy for the improvement!

Reply:
Yeah, it's because it's a different application. I think at some point we switched to running each application eunit test in a new process for some reason. I wonder if it's possible to switch back so we can get the unified coverage report. For instance chttpd tests would generate good coverage but we just won't see it, as we run the tests today.